parm/post_avblflds.xml

```diff
@@ -6635,6 +6635,20 @@
        <scale>3.0</scale>
     </param>
 
+    <param>
+       <post_avblfldidx>712</post_avblfldidx>
+       <shortname>AER_OPT_AQM_at550</shortname>
+       <pdstmpl>tmpl4_48</pdstmpl>
+       <pname>AOTK</pname>
+       <stats_proc>AVE</stats_proc>
```

Comment:
@WenMeng-NOAA It looks the current output aod is instant. value instead of hourly average. So we may need to remove AVE here?

Reply:
@KaiWang-NOAA Got it. Will update.